src/saml/User.php

```diff
@@ -18,23 +18,32 @@ public static function convertToSamlFieldNames(
         $profileReview,
         array $member
     ) {
+
+        // eduPersonUniqueId (only alphanumeric allowed)
+        $epuid = str_replace('-', '', $uuid);
+        $epuid .= '@' .$idpDomainName;
```

Comment:
Would you be okay with us using non-abbreviated variable names here, to improve clarity?
```suggestion
        $alphaNumericUuid = str_replace('-', '', $uuid);
        $eduPersonUniqueId = $alphaNumericUuid . '@' . $idpDomainName;
```